### ISSUE TITLE
Ability to change SECRET_KEY

### ISSUE THREAD

**Issue by tiredofit** [logchimp/logchimp#175]
As I understand, Secret Key allows for crypt/decrypt of password hashes. It would be ideal to be able to have some sort of routine to be able to rotate that key at some point in the event of a security concern.

My thoughts would be that it could be an argument via <command> swap-secret <oldkey> <newkey> to perform necessary database hashing, while updating the config file either manually or automatically after the fact.

**Comment by mittalyashu** [comment]
We're working on a feature in [LogChimp CLI](https://www.npmjs.com/package/logchimp-cli) where you can update the logchimp configuration file using with command

```bash
$ logchimp comfig:set --key server.secretKey --value newSecretKey
```

You have a valid point to have the option for rotating the secretKey for security reason, but changing the existing secretKey would mean that the existing user might not able to login into their account and need to reset the password.

> argument via swap-secret to perform necessary database hashing

Can you share more detail about this?

**Comment by tiredofit** [comment]
You've covered it with the `config:set` idea. That's exactly what I was trying to articulate..

**Comment by mittalyashu** [comment]
This feature will be release in v0.2.0 of [LogChimp CLI](https://github.com/logchimp/logchimp-cli)

Related to this commit https://github.com/logchimp/logchimp-cli/commit/dfae5b3b72876f1b5d6ea9ff09d2d05bb416fe9d

**Comment by mittalyashu** [comment]
Closing this issue, this feature was released in [LogChimp CLI v0.2.0](https://github.com/logchimp/logchimp-cli/releases/tag/v0.2.0).
